src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const totalToPay = rentCost * days;
+
+  if (days >= 7) {
```

Comment:
The conditional value (7) here is still being hardcoded, which is not a good practice